src/client/Microsoft.Identity.Client/Internal/Requests/InteractiveRequest.cs

```diff
@@ -286,7 +286,7 @@ private void VerifyAuthorizationResult()
 
         private async Task<MsalTokenResponse> ExecuteBrokerAsync(CancellationToken cancellationToken)
         {
-            IBroker broker = base.ServiceBundle.PlatformProxy.CreateBroker();
+            IBroker broker = base.ServiceBundle.PlatformProxy.CreateBroker(_interactiveParameters.UiParent);
```

Comment:
Yes, this is a better design - much easier to understand that the broker needs to know about the UI parent.